teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>typo: fixing <strong>the</strong> protein</p><p> </p><p>Why is fixing the ligand a difficult task? You could add one or two sentences that it can be hard to determine the exact geometry from a pdb file, since bonding information is sometimes incomplete or erroneous and that hydrogens are often not present in pdb structures.</p><p> </p><p>typo: which <strong>is specialized</strong> in small molecules</p><p> </p><p>typo: <strong>so</strong> we need to assign</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>The missing hydrogens are mentioned in the protein fix and the missing bond-orders here but I tried to emphasize it more. </p><p>And I only labeled it as "different" not more difficult. Is it better described as difficult?</p>